README.md

```diff
@@ -1026,6 +1027,42 @@ datetime                   inExchanges  inTopHoldersTotal  outsideExchanges
 2019-04-11 00:00:00+00:00     7.975260          13.296356          5.321096
 ```
 
+### News
+
+Returns the news for given word.
+
+Arguments description:
+
+  *  tag - Project name, ticker or other crypto related words.
```

Comment:
nitpick: only `tag`'s description starts with a capital letter

Reply:
Made it into a lowercase letter